dask_expr/expr.py

```diff
@@ -404,6 +404,18 @@ def min(self, skipna=True, numeric_only=False, min_count=0):
     def count(self, numeric_only=False):
         return Count(self, numeric_only)
 
+    def unique(self):
+        # let pandas raise errors, e.g. not defined on DataFrame
+        self._meta.unique()
+        return Unique(self)
+
+    def drop_duplicates(self, subset=None, ignore_index=False):
+        if subset is not None:
+            # Fail early if subset is not valid
+            meta_nonempty(self._meta).drop_duplicates(subset=subset)
+
+        return DropDuplicates(self, subset=subset, ignore_index=ignore_index)
```

Comment:
If these are only on `Series/DataFrame` then maybe the answer is that we don't put them on `Expr`, but instead only put them on `Series/DataFrame`.

See https://github.com/mrocklin/dask-expr/issues/98#issuecomment-1556191825 for this same suggestion on a larger issue.

Reply:
Ah, I forgot that we already had this conversation on another PR as well.  Thoughts here?

Reply:
Yep makes sense, will update

Reply:
Updated